apps/sms/views/conversation/js/conversation.js

```diff
@@ -79,6 +84,13 @@ var ConversationView = {
     preventLastCall: false
   },
 
+  LATE_NOTICE_LOWER_LIMITS: {
+    NO_NOTICE: 1 * MINUTE,
```

Comment:
I feel we should not display it here for less than 5 minutes. Because it's quite current to have desynchronized clocks (for example, my phone has a 2-minute difference from my girlfriend's phone !)